web/index.html

```diff
@@ -17,12 +17,12 @@
 		<div id="wrapper">
 			<div id="content">
 				<h1>Vircadia Community Applications Repository</h1>
-				<p>This repository stores optional <b>tablet applications</b> and <b>functionalities</b> provided by the contributions of the community members. 
-				If you have created an Application or functionality script and want to offer it to the community, this is where you can do it.</p>
+				<p>This repository stores optional <b>tablet applications</b> provided by the contributions of community members who are not core developers of Vircadia. 
+				If you have created a tablet application and want to offer it to the community, this is where you can do it.</p>
```

Comment:
```suggestion
				If you have created an application and want to offer it to the community, this is where you can do it.</p>
```
Like talked about; You could have any kind of script in here. Even without a dedicated tablet app button.
I *think* Basinskys VRGrabScale app doesn't use a tablet icon, but I am not sure.

Reply:
I kind of like the clarity of "tablet application", just so people wouldn't wonder if they can upload applications they have written for other platforms. Either way is good though. :)